EnglishFilter/sections/css_extended.txt

```diff
@@ -1,6 +1,8 @@
 !
 ! Extended CSS rules
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/80140
+tellerreport.com#?#article.flex-column > div[class]:contains(/^ADS$/):upward(2)

```

Comment:
or `tellerreport.com#?#.align-content-stretch > div.flex-fill:has(> article.flex-column > div[class=""]:contains(/^ADS$/))`

Reply:
I prefer mine more. You're one is not long lasting.